requirements-dev.txt

```diff
@@ -5,3 +5,4 @@ isort==5.10.1
 pytest==7.0.0
 pytest-cov==3.0.0
 pytest-mock==3.7.0
+google-api-python-client==2.46.0
```

Comment:
why do we need this?

Reply:
Otherwise I have this error:
![error](https://user-images.githubusercontent.com/45267439/166261644-56eb57cd-72b4-4dd7-84ea-ef540857ac52.png)

Reply:
Needed during `da.push()`

Reply:
docarray did not install it by default?

Reply:
should it be in `dev` though?

Reply:
I'll ask about it on slack

Reply:
I asked for help on the docarray channel. So maybe we can ignore it for now and I'll revisit that later.

Reply:
@gmastrapas 
> should it be in dev though?'

If it's needed during `da.push()`, we should have it in `requirements.txt`, good point.